IBX-7076: Customizable dashboard described in User Documentation (#246)
ibexa/documentation-user: docs/getting_started/dashboard/dashboard.md

---
description: Get to know about dashboard.
---

# Dashboard

> Dashboard it the first screen that you see after you log into [[= product_name =]].
> By default, it contains the following blocks:

Reviewer: ```suggestion
Dashboard is the first screen that you see after you log into [[= product_name =]].
By default, it contains the following blocks:
```

- Quick actions
- Recent activity
- Ibexa news
- Review queue
- My content
- Common content

> 

Author: This information is not needed. 
It is covered in these places: 
- "By default, it contains the following blocks (...)"
- "Dashboard is the first screen that you see after you log into (...)"

![Dashboard](dashboard.png "Dashboard")

For a detailed description of all available blocks, see [Dashboard block reference](dashboard_block_reference.md).

> You can return to the **Dashboard** screen from any other page by clicking the '[[= product_name_base =]]' logoon the left of the top bar, or the home icon in the main menu.

Reviewer 2: ```suggestion
You can return to the **Dashboard** screen from any other page by clicking the '[[= product_name_base =]]' logo on the left of the top bar, or the home icon in the main menu.
```

> You can return to the **Dashboard** screen from any other page by clicking the '[[= product_name_base =]]' logo on the left of the top bar, or the home icon in the main menu.

Reviewer 3: ```suggestion
You can return to the **Dashboard** screen from any other page by clicking the [[= product_name_base =]] logo on the left of the top bar, or the home icon in the main menu.
```
